fastlane/Fastfile

```diff
@@ -585,6 +585,31 @@ platform :ios do
     )
   end
 
+  # Upload the localized metadata (from `fastlane/metadata/`) to App Store Connect
+  #
+  # @option [Boolean] with_screenshots (default: false) If true, will also upload the latest screenshot files to ASC
+  #
+  desc 'Upload the localized metadata to App Store Connect, optionally including screenshots.'
+  lane :update_metadata_on_app_store_connect do |options|
+    # Skip screenshots by default. The naming is "with" to make it clear that
+    # callers need to opt-in to adding screenshots. The naming of the deliver
+    # (upload_to_app_store) parameter, on the other hand, uses the skip verb.
+    with_screenshots = options.fetch(:with_screenshots, false)
+    skip_screenshots = !with_screenshots
+
+    upload_to_app_store(
+      app_identifier: APP_STORE_VERSION_BUNDLE_IDENTIFIER,
+      app_version: ios_get_app_version,
+      skip_binary_upload: true,
+      screenshots_path: File.join(FASTLANE_DIR, 'promo_screenshots'),
+      skip_screenshots: skip_screenshots,
+      overwrite_screenshots: true, # won't have effect if `skip_screenshots` is true
+      phased_release: true,
+      precheck_include_in_app_purchases: false,
+      api_key_path: ASC_KEY_PATH
```

Comment:
One question I had was related to the `api_key_path`: the `Deliverfile` originally had `.configure-files/app_store_connect_fastlane_api_key.json`, but the existing constant `ASC_KEY_PATH` has a completely different path to the same file, which is the one I'm using here.
Perhaps someone can shed some light on the reasoning behind the two different paths? Perhaps the files are redundant?

Reply:
> The best explanation that I have is that they had that file in their machine back from the time they were doing releases before #4576 landed (so back when that path was correct), and since those files are in .gitignore and they never ran git clean to remove those old files from their Mac, they happen to still have those files from before around (and since the API key hasn't changed since… the old file was still valid) and that's why it didn't break
That's some serious 🕵️ work right there 😄 but yes, that would explain it, it makes sense and is a very good guess.